### ISSUE TITLE
!Error in pak subprocess

### ISSUE THREAD

**Issue by YushufSharker** [posit-dev/shinychat#12]
The installation of Shinychat stopped due to an error I do not understand. I have included the results of the pak command

> pak::pak("jcheng5/shinychat")
✔ Updated metadata database: 5.50 MB in 15 files.                         
✔ Updating metadata database ... done                                     
                                                                            
→ Will install 9 packages.
→ Will download 8 CRAN packages (9.68 MB).
→ Will download 1 package with unknown size.
+ commonmark    1.9.2      [dl] (142.64 kB)
+ coro          1.0.5      [dl] (149.48 kB)
+ httpuv        1.6.15     [dl] (1.00 MB)
+ later         1.3.2      [dl] (448.58 kB)
+ promises      1.3.0      [dl] (2.01 MB)
+ shiny         1.9.1      [dl] (5.09 MB)
+ shinychat     0.0.0.9000 [bld][cmp][dl] (GitHub: ed766f2)
+ sourcetools   0.1.7-1    [dl] (128.03 kB)
+ xtable        1.8-4      [dl] (710.58 kB)
ℹ Getting 8 pkgs (9.68 MB) and 1 pkg with unknown size
✔ Got shinychat 0.0.0.9000 (source) (360.80 kB)                                                                                  
✔ Got commonmark 1.9.2 (x86_64-w64-mingw32) (142.65 kB)                          
✔ Got coro 1.0.5 (i386+x86_64-w64-mingw32) (149.72 kB)                           
✔ Got httpuv 1.6.15 (x86_64-w64-mingw32) (1.00 MB)                               
✔ Got later 1.3.2 (x86_64-w64-mingw32) (448.18 kB)                               
✔ Got sourcetools 0.1.7-1 (x86_64-w64-mingw32) (128.00 kB)                       
✔ Got xtable 1.8-4 (i386+x86_64-w64-mingw32) (711.67 kB)                         
✔ Got promises 1.3.0 (x86_64-w64-mingw32) (2.01 MB)                              
✔ Got shiny 1.9.1 (i386+x86_64-w64-mingw32) (5.09 MB)                            
✔ Downloaded 9 packages (10.04 MB) in 5.5s                                       
✔ Installed commonmark 1.9.2  (2.8s)                                                                                     
✔ Installed coro 1.0.5  (2.5s)                                                                                         
✔ Installed httpuv 1.6.15  (3.2s)                                                                                    
✔ Installed later 1.3.2  (3.5s)                                                                                
✔ Installed xtable 1.8-4  (3s)                                                                         
✔ Installed promises 1.3.0  (4.1s)                                                              
✔ Installed sourcetools 0.1.7-1  (3.9s)                                                 
✔ Installed shiny 1.9.1  (6.3s)                                               
ℹ Packaging shinychat 0.0.0.9000                                   
✔ Packaged shinychat 0.0.0.9000 (6.3s)                               
ℹ Building shinychat 0.0.0.9000                                      
Error:                                                               
! error in pak subprocess
Caused by error: 
! Could not find tools necessary to compile a package
Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
Type .Last.error to see the more details.

**Comment by jcheng5** [comment]
I think you may need to install the appropriate version of RTools: https://cran.r-project.org/bin/windows/Rtools/ Sorry for the inconvenience. This won’t be necessary once shinychat goes to CRAN.

**Comment by YushufSharker** [comment]
Yes, You are right! Updating Rtools solved my problem. Thanks so much!
